### ISSUE TITLE
QIO Storage Port, QIO Dashboard Box and QIO Dashboard upgrades.

### ISSUE THREAD

**Issue by iiiCpu** [mekanism/Mekanism-Feature-Requests#137]
**QIO Storage Port** (like **AE2**'s **_ME Storage Bus_**) allows to interact with contents of connected chests without importing it into drive. Might be useful in early QIO for storing equipment, especially with "**Iron Chests**"-like mods.

**QIO Dashboard Box** is opposite of **_QIO Storage Port_** - it shows all saved contents like usual chest, allowing connections with hoppers (both in and out), Mechanical pipes and other mods. Just for **AE2**'s **_ME Storage Bus_**, obviously.

QIO Dashboard upgrades:
- **Crafting Card**- adds **_Crafting Table_** with automatic refilling
- **Quick Item Access** - RMB take a stack of selected item, Shift+RMB open dashboard
- **Filter Card** - applies before **_QIO Dashboard search_**, allows advanced filtering by text, type (food, tool, equipment, ore, others), enchantment (usual\enchanted items), durability level (perfect\above\below %). Also may be used on QIO Exporter and Importer.
 - **Flash Card** - adds small separate inventory

Also, MekaSuit QIO Upgrades:
 - **Speedloader** (Bodyarmor) - refills items in inventory from linked QIO Storage. Does not fill empty cells.
 - **TeleCleaner** (Boots) - sends all gathered items directly to QIO Storage, even (and especially) if inventory is full.

**Comment by iiiCpu** [comment]
Oh, one more thing. Can you make **QIO Dashboard** Search box to lose focus? For JEI purpose.

**Comment by 7eggert** [comment]
I don't think that having an interface for e.g. Integrated Dynamics would be that bad: ID can handle Collossal Chests and they are even more nighmar-ish than a storage system with infinite stack size would be (because Collossal Chests are just chests with a lot of slots). But if you want to be sure: Make a tooltip "\n(This is expected to be SLOW,\n use importers/exporters if possible)"
